pkg/processor/runtime/python/runtime.go

```diff
@@ -229,6 +232,7 @@ func (py *python) handleResponseLog(functionLogger nuclio.Logger, response []byt
 	log := make(map[string]interface{})
 
 	if err := json.Unmarshal(response, &log); err != nil {
+		functionLogger.ErrorWith("Can't decode log", "error", err)
```

Comment:
functionLogger should only be used for records that are of interest to the user. That is, if it's the user's fault - it goes to functionLogger. Otherwise it goes to the resource logger (this case - runtime).

Reply:
OK